tests/integration/components/questionnaire/exomeseq-studytype-choice-test.js

```diff
@@ -0,0 +1,67 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+import Ember from "ember";
+
+moduleForComponent('questionnaire/exomeseq-studytype-choice', 'Integration | Component | questionnaire/exomeseq studytype choice', {
+  integration: true
+});
+
+test('it renders heading label', function(assert) {
+  this.render(hbs`{{questionnaire/exomeseq-studytype-choice}}`);
+  assert.equal(this.$('.study-type.heading').text().trim(), 'Choose the type of your study:');
+});
+
+test('it renders choices with descriptions', function(assert) {
+  this.render(hbs`{{questionnaire/exomeseq-studytype-choice}}`);
+  assert.equal(this.$('.study-type.choice').eq(0).text().trim(), 'Small Familial');
+  assert.equal(this.$('.study-type.choice input').eq(0).attr('value'), 'Small Familial');
+  assert.equal(this.$('.study-type.choice').eq(1).text().trim(), 'Large Population (Requires 20+ samples)');
+  assert.equal(this.$('.study-type.choice input').eq(1).attr('value'), 'Large Population');
+});
+
+test('it sends answerChanged when selecting a choice', function(assert) {
+  const expectedValues = ['Small Familial', 'Large Population'];
+  assert.expect(expectedValues.length);
+  expectedValues.forEach((expectedValue, index) => {
+    this.set('answerChanged', (sender) => {
+      assert.equal(sender.get('answerValue'), expectedValue);
+    });
+    this.render(hbs`{{questionnaire/exomeseq-studytype-choice answerChanged=answerChanged}}`);
+    this.$('.study-type.choice').eq(index).click();
+  });
+});
+
+// Below are copied verbatim from string-field-test, suggesting a refactoring should be done
```

Comment:
Perhaps create an issue to deal with this later. Right now AnswerableField subclasses must include something like this in their hbs: 
```
{{#if answerFormErrors.show}}
    {{error-panel errors=fieldErrors}}
{{/if}}
```

Reply:
Created #119 to address duplication